adding BCC to Mail

meyer-solutions opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have submitted a [CLA form](https://github.com/IBM-Swift/CLA)
- [ ] If applicable, I have updated the documentation accordingly.
- [ ] If applicable, I have added tests to cover my changes.

meyer-solutions committed:
adding BCC to Mail

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Kitura/Swift-SMTP?pullRequest=133) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Kitura/Swift-SMTP?pullRequest=133) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Kitura/Swift-SMTP?pullRequest=133) it.</sub>

meyer-solutions committed:
Use LoggerAPI debug logging instead of print in DataSender

This gives users of Swift-SMTP more control over stdout output.
@jbmorley
jbmorley authored on Apr 3, 2025

meyer-solutions committed:
Remove BCC from email dictionary construction

Removed BCC field from email dictionary if empty.
